### ISSUE TITLE
[Feature Request] Scan Steam shadercache folders for entries

### ISSUE THREAD

**Issue by NoXPhasma** [rcpoison/dxvk-cache-pool#18]
When using Proton inside of Steam, the state cache files will be created in `$STEAMLIBRARYPATH/steamapps/shadercache/$GAMEID/DXVK_state_cache/`. Would be actually good and smart to scan those folders for cache files as well.

Also keep in mind that there are possible several game library folders in Steam. Those can be found in the `~/.steam/steam/config/config.vdf` file. Named `BaseInstallFolder_1`, `BaseInstallFolder_2` and so on.

**Comment by rcpoison** [comment]
Yes, I did consider adding support for steam but since valve was pretty quick moving the .dxvk files to the shader cache dir I assumed they'd be quick implementing sharing too and didn't want to interfere with their sharing method.

Locating the shader directory sadly isn't that straightforward especially for steam directories that existed for a few years and became quite messy with cyclic symlinks and alternative dir names. 

Plus I'd need to implement special handling since steam overrides the dxvk target and I couldn't figure out a sane way to override their override.

If you volunteer as tester I'll implement it ;)

**Comment by NoXPhasma** [comment]
Of course I'm willing to test it for you. :)
